specification/eventgrid/data-plane/Microsoft.Communication/stable/2018-01-01/AzureCommunicationServices.json

```diff
@@ -502,6 +502,82 @@
         }
       }
     },
+    "AcsEmailDeliveryReportReceivedEventData": {
+      "description": "Schema of the Data property of an EventGridEvent for a Microsoft.Communication.EmailDeliveryReportReceived event.",
+      "type": "object",
+      "properties": {
+        "sender": {
+          "description": "The Sender Email Address",
+          "type": "string"
+        },
+        "recipient": {
+          "description": "The recipient Email Address",
+          "type": "string"
+        },
+        "messageId": {
+          "description": "The Id of the email been sent",
+          "type": "string"
+        },
+        "status": {
+          "description": "The status of the email",
+          "type": "string",
+          "enum": [
+            "Failed",
+            "Delivered",
+            "FilteredSpam",
+            "Quarantined"
+          ],
+          "x-ms-enum": {
+            "name": "AcsEmailDeliveryReportStatus",
+            "modelAsString": true
+          }
+        },
+        "deliveryAttemptTimestamp": {
```

Comment:
As mentioned [here](https://github.com/Azure/azure-rest-api-specs/pull/22272#discussion_r1101894622), we can't actually change the property name since these events are already live. Instead we can use x-ms-client-name to add the appropriate casing.